pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -41,21 +40,13 @@ def __init__(self) -> None:
         self.manual_loop = ManualOptimization()
 
         self._warning_cache: WarningCache = WarningCache()
-        self._optimizer_freq_cumsum: Optional[int] = None
         self._remaining_splits: Optional[List[Any]] = None
 
     @property
     def done(self) -> bool:
         """Returns if all batch splits have been processed already."""
         return len(self._remaining_splits) == 0
 
-    @property
-    def optimizer_freq_cumsum(self) -> int:
-        """Returns the cumulated sum of optimizer frequencies."""
-        if self._optimizer_freq_cumsum is None:
-            self._optimizer_freq_cumsum = np.cumsum(self.trainer.optimizer_frequencies)
```

Comment:
this value no longer gets cached. we recompute it every epoch because now the function is a utility. is this a problem?

Reply:
If we still want to cache it, we could add an `@lru_cache` decorator

Reply:
I tried but I run into the issue that list inputs can't be cached because they are not hashable

Reply:
yeah... would need to use a tuple, up to you.